coulomb-physical-constants/src/main/scala/coulomb/physicalconstants/package.scala

```diff
@@ -0,0 +1,495 @@
+
+  def newtonianConstantOfGravitation[V](implicit pcq: PhysicalConstantQuantity[V, NewtonianConstantOfGravitation]): Quantity[V, pcq.QU] = pcq.q
+
+  def vacuumElectricPermittivity[V](implicit pcq: PhysicalConstantQuantity[V, VacuumElectricPermittivity]): Quantity[V, pcq.QU] = pcq.q
+
+  def vacuumMagneticPermeability[V](implicit pcq: PhysicalConstantQuantity[V, VacuumMagneticPermeability]): Quantity[V, pcq.QU] = pcq.q
+
+  def characteristicImpedanceOfVacuum[V](implicit pcq: PhysicalConstantQuantity[V, CharacteristicImpedanceOfVacuum]): Quantity[V, pcq.QU] = pcq.q
+
+  def elementaryCharge[V](implicit pcq: PhysicalConstantQuantity[V, ElementaryCharge]): Quantity[V, pcq.QU] = pcq.q
+
+  def hyperfineTransitionFrequencyOfCs133[V](implicit pcq: PhysicalConstantQuantity[V, HyperfineTransitionFrequencyOfCs133]): Quantity[V, pcq.QU] = pcq.q
+
+  def avogadroConstant[V](implicit pcq: PhysicalConstantQuantity[V, AvogadroConstant]): Quantity[V, pcq.QU] = pcq.q
+
+  def boltzmannConstant[V](implicit pcq: PhysicalConstantQuantity[V, BoltzmannConstant]): Quantity[V, pcq.QU] = pcq.q
+
+  def conductanceQuantum[V](implicit pcq: PhysicalConstantQuantity[V, ConductanceQuantum]): Quantity[V, pcq.QU] = pcq.q
+
+  def josephsonConstant[V](implicit pcq: PhysicalConstantQuantity[V, JosephsonConstant]): Quantity[V, pcq.QU] = pcq.q
+
+  def vonKlitzingConstant[V](implicit pcq: PhysicalConstantQuantity[V, VonKlitzingConstant]): Quantity[V, pcq.QU] = pcq.q
+
+  def magneticFluxQuantum[V](implicit pcq: PhysicalConstantQuantity[V, MagneticFluxQuantum]): Quantity[V, pcq.QU] = pcq.q
+
+  def inverseConductanceQuantum[V](implicit pcq: PhysicalConstantQuantity[V, InverseConductanceQuantum]): Quantity[V, pcq.QU] = pcq.q
+
+  def bohrMagneton[V](implicit pcq: PhysicalConstantQuantity[V, BohrMagneton]): Quantity[V, pcq.QU] = pcq.q
+
+  def nuclearMagneton[V](implicit pcq: PhysicalConstantQuantity[V, NuclearMagneton]): Quantity[V, pcq.QU] = pcq.q
+
+  def fineStructureConstant[V](implicit pcq: PhysicalConstantQuantity[V, FineStructureConstant]): Quantity[V, pcq.QU] = pcq.q
+
+  def inverseFineStructureConstant[V](implicit pcq: PhysicalConstantQuantity[V, InverseFineStructureConstant]): Quantity[V, pcq.QU] = pcq.q
+
+  def electronMass[V](implicit pcq: PhysicalConstantQuantity[V, ElectronMass]): Quantity[V, pcq.QU] = pcq.q
+
+  def protonMass[V](implicit pcq: PhysicalConstantQuantity[V, ProtonMass]): Quantity[V, pcq.QU] = pcq.q
+
+  def neutronMass[V](implicit pcq: PhysicalConstantQuantity[V, NeutronMass]): Quantity[V, pcq.QU] = pcq.q
+
+  def bohrRadius[V](implicit pcq: PhysicalConstantQuantity[V, BohrRadius]): Quantity[V, pcq.QU] = pcq.q
+
+  def classicalElectronRadius[V](implicit pcq: PhysicalConstantQuantity[V, ClassicalElectronRadius]): Quantity[V, pcq.QU] = pcq.q
+
+  def electronGFactor[V](implicit pcq: PhysicalConstantQuantity[V, ElectronGFactor]): Quantity[V, pcq.QU] = pcq.q
```

Comment:
could add the negation here, just add a comment about it

Reply:
you will probably also have to add implicits to specify that `V` is AdditiveGroup.  I think that will quality it as negatable.  If not that, there will be a way.

Reply:
It seems I can make it work with `UnitNeg` as:
`def electronGFactor[V: UnitNeg](implicit pcq: PhysicalConstantQuantity[V, ElectronGFactor]): Quantity[V, pcq.QU] = -pcq.q`

Reply:
aha yes, even better